client/src/Components/EmployeeProfile/index.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import './EmployeeProfile.scss';
+import Header from '../Header/Header'
```

Comment:
Here you are building 2 components together. When you are working on the EmployeeeProfile component just work on that. You are building heard as well here which 1 is not a good practice and 2. there is no header component in our backlog, we have a navigation component which i am doing. so you don't have to worry about how the header will look like. just do employee profile here. :)